internal/pkg/scorecard/scorecard.go

```diff
@@ -394,15 +89,20 @@ func ScorecardTests(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	cmd.SilenceUsage = true
-	pluginOutputs, err := runTests()
+	var err error
```

Comment:
Why the error declaration before?

Reply:
On the next line, we are setting a global variable, so we have to use `=`, which means that `err` needs to be declared. If we use `:=`, that creates a new local variable and doesn't change the global one.

Reply:
Interesting! Could we add a comment? So someone doesn't remove that in the future as it's not super obvious, at least not to me.